app/controllers/posts_controller.rb

```diff
@@ -2,11 +2,25 @@ class PostsController < ApplicationController
   def index
```

Comment:
- [ ] You should use **CanCanCan** for Post authorization in `Post Controller`
example:

```ruby
 class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```